Bio/PDB/Chain.py

```diff
@@ -94,6 +94,12 @@ def _translate_id(self, id):
          - id - int, residue resseq
 
         """
+        try:
+            id = int(id)
```

Comment:
This does not seem to accept octal or hex notation (without explicitly giving the base), which is good here.